scripts/linters/css_linter_test.py

```diff
@@ -65,8 +70,10 @@ def mock_join(*unused_args):
             'node-stylelint and its dependencies.'):
             third_party_linter.perform_all_lint_checks()
 
-    def test_perform_all_lint_checks_with_stderr(self):
-        def mock_popen(unused_commands, stdout, stderr):  # pylint: disable=unused-argument
+    def test_perform_all_lint_checks_with_stderr(self) -> None:
+        def mock_popen(
+            unused_commands: List[str], stdout: int, stderr: int  # pylint: disable=unused-argument
```

Comment:
This seems incorrect to me, I don't think stdout and stderr are typically represented as ints.

Reply:
Ah, thanks for checking. OK sounds good, can you please just put a comment here saying something like "Note that, in general, stdout and stderr can be ints -- e.g. subprocess.PIPE is an Integer constant whose value is -1. Reference: https://github.com/python/cpython/blob/8827b95e80302ef19d19561fdcd81c6efde2ecdb/Lib/subprocess.py#L259" ? That will probably help other devs.

Thanks!

Reply:
Comment added!